connector/mqtt/latency-pub-test-prometheus/app/Utils.js

```diff
@@ -0,0 +1,15 @@
+const extractPayload = (message) => {
+  const messageContent = JSON.parse(message);
+  const { attrs } = messageContent;
```

Comment:
Only to reduce line of code, can it be?
return messageContent.attrs?

Reply:
https://github.com/airbnb/javascript#destructuring